modules/fedimint-ln-common/src/db.rs

```diff
@@ -57,6 +58,51 @@ impl_db_lookup!(
     query_prefix = ContractUpdateKeyPrefix
 );
 
+/// We keep a separate mapping of incoming and outgoing ContractIds to Amounts,
+/// which allows us to quickly audit the total liabilities in the Lightning
+/// module.
+///
+/// This differs from MintAuditItemKeys, since it doesn't include an aggregate
+/// *Total key. The motivation for not including the aggregate key is how the
+/// Amount associated to the contract mutates in the LN module. When a contract
+/// reaches a terminal state, the associated amount updates to 0. The additional
+/// complexity to update both the individual incoming/outgoing contract audit
+/// keys along with the aggregate audit key is not necessary.
+//
+// In contrast to the mint module, the total number of LN audit keys with a non-zero amount will not
+// grow linearly, so querying the LN audit keys with a non-zero amount should remain quick.
```

Comment:
Is the missing `/` intentional, i.e. you don't want this to be displayed in the docs?

Reply:
Good eye! That was unintentional :smiley: